.circleci/config.yml

```diff
@@ -78,43 +90,42 @@ jobs:
 
-      - run:
-          name: Build plugin artifact
-          command: ./gradlew buildPlugin
-      - store_artifacts:
-          path: build/distributions/
-          destination: .
-      - run:
-          name: Verify plugin artifact contents
-          command: ./scripts/verify-artifact-contents
-      - run:
-          name: Verify binary compatibility with supported IntelliJ versions
-          command: ./gradlew runPluginVerifier
+    #  - run:
+    #      name: Build plugin artifact
+    #      command: ./gradlew buildPlugin
+    #  - store_artifacts:
+    #      path: build/distributions/
+    #      destination: .
+    #  - run:
+    #      name: Verify plugin artifact contents
+    #      command: ./scripts/verify-artifact-contents
+    #  - run:
+    #      name: Verify binary compatibility with supported IntelliJ versions
+    #      command: ./gradlew runPluginVerifier
 
+      - run:
+          name: Build ide-probe fork
+          # language=sh
+          command: |
+            cd ..
+            git clone https://github.com/azdrojowa123/ide-probe.git -b feature/screenshot_to_dir
+            cd ide-probe/
+            sbt api_2_13/publishM2 driver_2_13/publishM2 junit-driver_2_13/publishM2 robot-driver_2_13/publishM2 probe-plugin_2_13/publishM2
       - run:
           name: Run UI tests against an EAP of the latest supported major IntelliJ version (if applicable)
           command: ./gradlew -Pheadless -Pagainst=eapOfLatestSupportedMajor uiTest
       - run:
           name: Run UI tests against the latest stable IntelliJ version
           command: ./gradlew -Pheadless -Pagainst=latestStable uiTest
-      - when:
-          condition:
-            # Practice shows that the risk of UI tests failing on `latestMinorsOfOldSupportedMajors`
-            # once both binary compatibility checks and UI tests for latest versions passed is close to zero.
-            # Hence, to speed up the builds, we only run them for develop/master/hotfix/release and not on PRs.
-            matches:
-              pattern: "^(develop|hotfix/.+|master|release/.+)$"
-              value: << pipeline.git.branch >>
-          steps:
-            - run:
-                name: Run UI tests against earlier supported major IntelliJ versions
-                command: ./gradlew -Pheadless -Pagainst=latestMinorsOfOldSupportedMajors uiTest
+      - run:
+          name: Run UI tests against earlier supported major IntelliJ versions
+          command: ./gradlew -Pheadless -Pagainst=latestMinorsOfOldSupportedMajors uiTest
       - run:
           name: Collect artifacts when an exception occurs
           # language=sh
           command: |
             mkdir -p /exception-artifacts
-            for f in $(ls -d /tmp/ide-probe/screenshots/*/) ; do cd  $f/artifacts ; zip -r "/exception-artifacts/$(basename ${f}).zip" . ; done
+            for f in $(ls -d build/UIException/*/) ; do cd  $f/artifacts ; zip -r "/exception-artifacts/$(basename ${f}).zip" . ; done
```

Comment:
Actually... why `UIException`? 🤔 this name doesn't seem to appear anywhere else in the project

Reply:
I created UIException folder in` UITestSuite#saveThreadDumpAndScreenshot`